patches/kmc-v3.2.1_cuttlefish-dev.diff

```diff
@@ -0,0 +1,269 @@
+diff --git a/Makefile b/Makefile
+index bc07d35..2b351b9 100644
+--- a/Makefile
++++ b/Makefile
+@@ -13,15 +13,11 @@ OUT_BIN_DIR = bin
+ OUT_INCLUDE_DIR = include
+ 
+ ifeq ($(UNAME_S),Darwin)
+-	CC = /usr/local/bin/g++-10
+-
+-	CFLAGS	= -Wall -O3 -m64 -static-libgcc -static-libstdc++ -pthread -std=c++14
+-	CLINK	= -lm -static-libgcc -static-libstdc++ -O3 -pthread -std=c++14
++	CFLAGS	= -Wall -O3 -m64 -pthread -std=c++14
```

Comment:
Can we add `-fPIC` both to CFLAGS here and also in the non-Darwin branch? I think this may be required on the cluster.